### PR TITLE
Feature/xsd include

### PR DESCRIPTION
**Pull request by vvidovic**
Added implementation for the XSD include.

### CODE REVIEW

File: pkg/xsd/workspace.go [hunk 1]
```diff
@@ -40,9 +40,33 @@ func (ws *Workspace) loadXsd(xsdPath string) (*Schema, error) {
...
+		}
+
+		isch := si.IncludedSchema
+		schema.Imports = append(isch.Imports, schema.Imports...)
+		schema.Elements = append(isch.Elements, schema.Elements...)
+		schema.Attributes = append(isch.Attributes, schema.Attributes...)
+		schema.AttributeGroups = append(isch.AttributeGroups, schema.AttributeGroups...)
+		schema.ComplexTypes = append(isch.ComplexTypes, schema.ComplexTypes...)
+		schema.SimpleTypes = append(isch.SimpleTypes, schema.SimpleTypes...)
+		schema.inlinedElements = append(isch.inlinedElements, schema.inlinedElements...)
```
Comment (isimluk) [inline pkg/xsd/workspace.go:64]: Looking at these list appends.

Do we want to de-duplicate the elements? I mean may it happen that the current schema re-defines field of the  included schema?

**Review by isimluk — APPROVED** [review]
@vvidovic, looks fine! I am sorry it took me so long to do the review.

**Comment by isimluk** [comment]
@vvidovic, would you be willing to create a test for this?

I am fine, if you are unable to create the test, but wanted to ask anyway ... before I merge.

File: pkg/xsd/workspace.go (hunk 1 above)
Comment (vvidovic) [reply pkg/xsd/workspace.go:64]: I believe that should not happen - XSD `include` definitions should work in the same way as if the definition is copy-pasted from the included XSD to the XSD which included it. The same situation should occur if we define the same element or type twice in a single XSD file.

**Pull request merged**